src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageId>Microsoft.Azure.Management.ResourceGraph</PackageId>
     <Description>Provides Resource Graph operations for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.ResourceGraph</AssemblyName>
-    <Version>0.10.0-preview</Version>
+    <Version>1.0.0</Version>
     <PackageTags>Azure;ResourceGraph;</PackageTags>
     <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```

Comment:
Please update `PackageReleaseNotes`
Please update version number in AssemblyInfo.cs

Reply:
Updated will push the new changes.